bin/sync.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+set -x
```

Comment:
I would use -e here, because the final rm hides the possible errors in the first steps.